src/core/callouts/qgscallout.cpp

```diff
@@ -57,6 +59,10 @@ void QgsCallout::initPropertyDefinitions()
     {
       QgsCallout::Orientation, QgsPropertyDefinition( "Orientation", QgsPropertyDefinition::DataTypeString, QObject::tr( "Callout curve orientation" ),  QObject::tr( "string " ) + "[<b>auto</b>|<b>clockwise</b>|<b>counterclockwise</b>]", origin )
     },
+    {
+      QgsCallout::Margins, QgsPropertyDefinition( "Margins", QgsPropertyDefinition::DataTypeString, QObject::tr( "Margins" ), QObject::tr( "string of four doubles '<b>top,right,bottom,left</b>' or array of doubles <b>[top, right, bottom, left]</b>" ) )
+    },
+    { QgsCallout::WedgeWidth, QgsPropertyDefinition( "WedgeWidth", QObject::tr( "Wedge width" ), QgsPropertyDefinition::DoublePositive, origin ) },
```

Comment:
@nyalldawson "Wedge" is intentional or typo for "Edge" ?

Reply:
It's intentional - it refers to the triangular shaped wedge which comes out of the label box